### PR TITLE
Rotate OrdersPage chevron on click

### LINKED ISSUE #676: UI: When orders on trade page maximized, chevron still points outward
The chevron (**<**) should be pointing right (**>**) instead of left when maximized.

Screenshot from 2020-03-02 12-30-15

### PR DESCRIPTION
**Pull request by W3stside**
Closes #676 

Rotates chevron on open/close state

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr713--dexreact.review.gnosisdev.com

File: src/components/TradeWidget/index.tsx [hunk 1]
```diff
@@ -247,39 +247,33 @@ const OrdersPanel = styled.div`
...
-      background-color: #ecf2f7;
-    }
+  &::before {
+    display: block;
+    content: ' ';
+    background: url(${arrow}) no-repeat center/contain;
+    height: 1.2rem;
+    width: 1.6rem;
+    margin: 0;
+    transform: ${({ $isOpen }): string => ($isOpen ? 'rotateY(180deg)' : 'rotateY(0)')};
```
Comment (anxolin) [inline src/components/TradeWidget/index.tsx]: ubermeganit: only the the degrees change, so just add the actual number

Comment (W3stside) [reply src/components/TradeWidget/index.tsx]: Don't need turn I don't think

Comment (W3stside) [reply src/components/TradeWidget/index.tsx]: nvm u right u right

**Pull request merged**